mcu_cfg.hjson

```diff
@@ -77,7 +77,10 @@
 
 
     interrupts: {
-        // always zero
+        plic_intr_src:           64,
```

Comment:
are you sure it makes sense to have it exposed to the user? I do not think the user can change it anyway

Reply:
My idea was to put it here in case it needs to be changed it can be done in the json file and not in the RTL files.

Reply:
but if you change it to 32 or 128 this won't change the RTL, so it is actually dangerous to expose it to the users